.github/workflows/e2e-tests.yml

```diff
@@ -70,3 +70,5 @@ jobs:
           # TODO - share release with the create-kubewarden-cluster action
           helm uninstall -n kubewarden kubewarden-controller
```

Comment:
just realized, shouldn't we uninstall kubewarden-defaults too?

Reply:
Added `kubewarden-defaults`